src/protocol_nmea.cpp

```diff
@@ -758,11 +758,11 @@ void nmea_GPSTimeToUTCTimeMsPrecision_ZDA_debug(char* a, int aSize, int &offset,
 
     // Check for irregular update timing
     int32_t cpuDtMs = cpuMs - lastCpuMs;
-    bool cpuDtMsGood = _ABS(cpuDtMs) < 30000;
+    bool cpuDtMsGood = _ABS(cpuDtMs) < 5000;
```

Comment:
Should these be attached to a constant/macro, so they aren't so magic?  is 5sec appropriate?  Or should an even smaller number be considered?  2.5s?  2? 1.5?

Reply:
This code is intended to be temporary and removed so I didn't invest as much time into it.